wordpress-mobile/WordPress-Android
# Setup architecture for day selection and epilogue for blogging reminders

Fixes #14752 

This PR introduces logic to switch between the 3 supported screen for blogging reminders - prologue, day selection and epilogue. There is only the primary button visible now on the day selection and epilogue. Based on this PR it should be easily possible to build the UI part of the day selection and of the epilogue screen since the methods are ready. 

To test:
- Turn on the blogging reminders flag (Me/App settings/Test feature configuration - BloggingRemindersFeatureConfig)
- Create a new post
- Notice the blogging reminders prologue screen is shown
- Click on "Set goals"
- Notice the empty screen with disabled button is shown


To test: 
- Turn on the blogging reminders flag (Me/App settings/Test feature configuration - BloggingRemindersFeatureConfig)
- Go to site settings 
- Click on "Blogging goals"
- Notice the empty screen with disabled button is shown

## Regression Notes
1. Potential unintended areas of impact
None

PR submission checklist:
